library/CHART-TEMPLATE/tests/common_library_labels_test.yaml

```diff
@@ -0,0 +1,44 @@
+suite: test labels helpers
+templates:
+  - templates/deployment.yaml # we could check the same for service and serviceAccount as well
+release:
+  name: my-release
+  namespace: my-namespace
+set:
+  licenseKey: test-license-key
+  cluster: test-cluster
+tests:
+  # We cannot test behaviour without overwriting this since the version chart often changes
+  - it: sets labels according to values
+    set:
+      labels:
+        helm.sh/chart: fixed
+    asserts:
+      - equal:
+          path: metadata.labels
+          value:
+            app.kubernetes.io/instance: my-release
+            app.kubernetes.io/managed-by: Helm
+            app.kubernetes.io/name: CHART-TEMPLATE
+            app.kubernetes.io/version: 1.16.0
+            helm.sh/chart: fixed
```

Comment:
it would be nice just to check if this label exist, just to have an independent test, but i guess that it cannot be done

Reply:
we can, but it has the chart version in it, therefore we would need to update the test each time 🤯  
```
{{- define "common.naming.chart" -}}
{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
{{- end }}
```